### DISCUSSION TITLE
Linear Regression R - Squared value

### DISCUSSION THREAD

**Discussion by malikergin** [pycaret/pycaret#2584] [General]
Correlation between target and features are so slow such as 0.30 etc.. 

In Pycaret  I've conducted ML algorithms and results are interesting. In Pycaret Linear Regression  the R-squared is 0.92 while in Minitab it is 0.19. 

Why these are different ? By the way, other algorithms such as RF, Extra Boosting Regressor  worked well.

**Comment by ngupta23** [comment]
Check the model coefficients to make sure that the same features are being used in both cases.
